x-pack/test/apm_api_integration/tests/metrics/memory/generate_data.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+import type { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+
+export const config = {
+  memoryTotal: 536870912, // 0.5gb
+  memoryFree: 94371840, // ~0.08 gb
+  cGroupMemoryLimit: 104857000, // ~0.10gb
+  cGroupMemoryUsage: 38547456, // 0,03gb
+};
```

Comment:
nit: wdyt of this?

```suggestion
const gigabytes = (value: numer) = value * 1024^3;

export const config = {
  memoryTotal: gigabytes(0.5), 
  memoryFree: gigabytes(0.08),
  cGroupMemoryLimit: gigabytes(0.1),
  cGroupMemoryUsage: gigabytes(0.03),
};
```

Reply:
Btw. why the weird sizes? can we do

```suggestion
const gigabytes = (value: numer) = value * 1024^3;

export const config = {
  memoryTotal: gigabytes(64), 
  memoryFree: gigabytes(5.5),
  cGroupMemoryLimit: gigabytes(8),
  cGroupMemoryUsage: gigabytes(1.5),
};
```

Reply:
I just picked the values from a random document. way better now
https://github.com/elastic/kibana/pull/145348/commits/14fadf419b53fc19b61e406bb9d71567aa0d03cd